_data/meltano/utilities/superset/apache.yml

```diff
@@ -0,0 +1,32 @@
+name: superset
+namespace: superset
+label: Apache Superset
+description: A modern, enterprise-ready business intelligence web application.
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/apache/superset
+variant: apache
+pip_url: apache-superset==1.5.0 markupsafe==2.0.1
+requires:
+  files:
+  - name: files-superset
+    variant: meltano
+settings:
+- name: home
+  value: $MELTANO_PROJECT_ROOT/.meltano/run/superset
+  description: The home directory where Superset will store data.
+- name: additional_dependencies
+  description: An space separated list of python dependencies to include as part of startup.
+    A list of database driver dependencies can be found here https://superset.apache.org/docs/databases/installing-database-drivers
+- name: load_examples
+  kind: boolean
+  value: false
+  description: A boolean whether to load Superset's example datasets and dashboards. Warning, this takes a few mins to load.
```

Comment:
I don't think it makes sense for this to be a setting and to run the command every time on initialization, as it's something a user would only want to do once. We're better off instructing them to run `meltano invoke superset load_examples` manually if they want this.